transformations.c

```diff
@@ -457,6 +472,45 @@ int tr_eval_string(struct sip_msg *msg, tr_param_t *tp, int subtype,
 			val->rs.s = _tr_buffer;
 			val->flags = PV_VAL_STR;
 			break;
+		case TR_S_DATE2UNIX:
+			if(!(val->flags&PV_VAL_STR))
+				goto error;
+
+			char *tz;
+			struct tm date_tm;
+
+			memcpy(_tr_buffer, val->rs.s, val->rs.len);
+			_tr_buffer[val->rs.len] = 0;
+
+			memset(&date_tm, 0, sizeof date_tm);
+
+			if (!strptime(_tr_buffer, "%a, %d %b %Y %H:%M:%S GMT", &date_tm)) {
+				LM_ERR("Failed to parse Date header field\n");
+				goto error;
+			}
+
+			tz = getenv("TZ");
+			setenv("TZ", "", 1);
+			tzset();
+			snprintf(_tr_buffer, TR_BUFFER_SIZE, "%ld", mktime(&date_tm));
```

Comment:
Let's change the printing to `%lld`, with cast to `(long long)` please, for ARM32 build.  See commit 25107836 for the reason.

Reply:
Done